pages/news/2020-08-02-portable-mixxx-setup.html

```diff
@@ -0,0 +1,85 @@
+title: DJ on the fly with Mixxx and Raspberry Pi
+author: dennis
+date: 2020-08-02 12:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown:"extra" %}
+
+*One of the things that make Mixxx special is that it works on more platforms that just Windows or macOS.*
+*Thanks to Mixxx being open-source, it's possible create "unusual" setups that the developers didn't expect - for example by running it on a low-cost Raspberry Pi single board computer combined with a touchscreen.*
+*It's always great to see what cool stuff our users do with Mixxx, so when we discovered a video of it on Reddit, we asked Dennis to write a blog post about his project.*
+
+{% include "video.html" with video_id="UZam2aeX3Nc" %}
+
+### Going portable
+
+Since the advent of small, portable netbook computers and single board computers I have been trying to downsize my DJ setup.
+Frustrated by power issues, lack of performance and audio quality I resorted to a bulky but solid Thinkpad, a cheap controller and my software of choice: Mixxx.
+
+This is a small story on how developments in both open source hard- and software as well as an well equiped "parts-bin" ends up in a somewhat specific but useable portable DJ setup using Mixxx and a Raspberry Pi.
+
+This was until I was commissioned to make a random sound file player for an exhibition by [De Player](https://www.deplayer.nl/).
+In a move that later proved to be somewhat overkill, I grabbed a Raspberry Pi 3 that had been collecting dust and wrote some scripts to play random audio files.
+
+Up until then I had only worked with the original Raspberry Pi and I was quite shocked at the greatly improved performance of the Pi model 3.
+Out of curiosity, a quick search of the Rasbian software repository brought up a match for "Mixxx"... but wait, would it actually run?
+
+### Gaining power
+
+To my surprise Mixx did run fine!
+With new found hope of building a "laptop-less" portable DJ setup I connected my USB sound-card and cheap controller only to find out the Raspberry Pi could not deliver enough current to the USB - no surprise there.
+
+Eager to keep the setup simple and portable, I did not want to deal with external powered USB hubs.
+To no avail I tried various cheap USB cards and after many more low power warnings and high latencies, my eye fell on a white label brand controller (Akiyama Pulsar) with build-in soundcard.
+This "Traktor ready" controller had been given to me over the previous owner's frustrations with the lack of mappings for any other software than Traktor.
+
+### Putting one and two together
+
+Being both a controller and audio interface, the Akiyama Pulsar reduced the overhead created by multiple USB devices (?) allowing the Raspberry Pi to run Mixxx smoothly with < 20ms latency and no underruns.
```

Comment:
Addressing the (?): perhaps he means that the RPi can't handle the on-the-wire traffic for two separate devices but can for one.

Reply:
Possible. Do you have a suggestion how to rephrase it?